### PR TITLE
Cge 166 db adv entity update

### PR DESCRIPTION
**Pull request by Jack11M**
Added String values to the AgeRange, renamed FROM_13_TO_14 to FROM_12_TO_14, fixed tests.
Renamed Lang to I18n, fixed tests.

### CODE REVIEW

**Review by drw85 — APPROVED** [review]
Looking good. I have one question but I approve.

File: src/main/java/com/hillel/items_exchange/model/Location.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
     @Enumerated(EnumType.STRING)
     @Column(name = "lang")
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/model/Location.java:25]: It's looks like you missed rename it into Column annotation.

File: src/main/java/com/hillel/items_exchange/model/enums/I18n.java [hunk 2]
```diff
@@ -3,14 +3,14 @@
 import lombok.Getter;
 
 @Getter
-public enum Lang {
-    UA("ua"),
-    EN("en"),
-    RU("ru");
+public enum I18n {
+    UA("UA_uk"),
+    EN("EN_en"),
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/model/enums/I18n.java:8]: Awesome! 🥇

**Review by rpkyrych — CHANGES_REQUESTED** [review]
Fix Column and check Sergiy's question.

File: src/main/java/com/hillel/items_exchange/model/Location.java (hunk 1 above)
Comment (Jack11M) [reply src/main/java/com/hillel/items_exchange/model/Location.java:25]: No, I have not changed the column name in the DB. We did not discuss this. Is it critical? I can do it, but you will need to remove the DB one more time.

File: src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java [hunk 3]
```diff
@@ -60,7 +59,7 @@ void findAll_shouldReturnAllLocationDtos() {
     void findById_shouldReturnLocation() {
         when(locationRepository.findById(anyLong())).thenReturn(Optional.of(location));
 
-        Location foundLocation = locationService.findById(1L).get();
+        Location foundLocation = locationService.findById(1L).orElseGet(Location::new);
```
[the remark this answers is not in the document]
Comment (Jack11M) [reply src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java:62]: As I know, it is a really bad solution to use Optional.get() directly without checking ifPresent. At these places we had such approach, so I decided to fix them

[the remark this answers is not in the document]
Comment (rpkyrych) [reply src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java:62]: You didn't understand the question: you used Mockito above with returning location from there if anyone triggers **.findById** method and here you make such triggering instead the test. So why do we need doing that?

File: src/main/java/com/hillel/items_exchange/model/Location.java (hunk 1 above)
Comment (rpkyrych) [reply src/main/java/com/hillel/items_exchange/model/Location.java:25]: Difference between names creates misunderstanding - you should update it or rename field name to lang.

Comment (Jack11M) [reply src/main/java/com/hillel/items_exchange/model/Location.java:25]: Ok

File: src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jack11M) [reply src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java:62]: I understood it. But findById returns Optional! And we just use get method, it is ugly in my opinion.

[the remark this answers is not in the document]
Comment (Jack11M) [reply src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java:62]: But I can return the previous code of you wish.
By the way, you told me that use Optional.get() unacceptable.

[the remark this answers is not in the document]
Comment (Jack11M) [reply src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java]: Fixed

File: src/main/java/com/hillel/items_exchange/model/Location.java (hunk 1 above)
Comment (Jack11M) [reply src/main/java/com/hillel/items_exchange/model/Location.java]: Done

File: src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java [hunk 4]
```diff
@@ -72,7 +71,7 @@ void findById_shouldReturnLocation() {
     void getById_shouldReturnLocationDto() {
         when(locationRepository.findById(anyLong())).thenReturn(Optional.of(location));
 
-        LocationDto foundLocation = locationService.getById(1L).get();
+        LocationDto foundLocation = locationService.getById(1L).orElseGet(LocationDto::new);
```
[the remark this answers is not in the document]
Comment (Jack11M) [reply src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java]: Fixed

File: src/main/java/com/hillel/items_exchange/model/Location.java [hunk 5]
```diff
@@ -22,8 +22,8 @@
     private String city;
 
     @Enumerated(EnumType.STRING)
-    @Column(name = "lang")
-    private Lang lang;
+    @Column(name = "i18n")
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/model/Location.java:25]: You could not to use annotation Column if the names of the column in DB and into entity are absolutely equal

**Review by rpkyrych — APPROVED** [review]

File: src/main/java/com/hillel/items_exchange/model/Location.java (hunk 5 above)
Comment (Jack11M) [reply src/main/java/com/hillel/items_exchange/model/Location.java]: Fixed

**Pull request merged**